standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -1639,21 +1562,6 @@ PartitionsResponse getPartitionsRequest(PartitionsRequest req)
         boolean getColStats, String engine, String validWriteIdList, Long tableId)
         throws TException;
```

Comment:
This method(line:1638) seems can be removed

Reply:
That was a tougher nut, but I hope I was successful 😄 
Let's see the test results